wwwroot/inc/snmp.php

```diff
@@ -4310,6 +4320,37 @@
 		'text' => 'HP 1810-8G v2 (J9802A)',
 		'processors' => array ('procurve-8ports-1000T'),
 	),
+	'11.2.3.7.11.151' => array
+	(
+		'dict_key' => 2242,
+		'text' => 'HP 1810-24G v2 (J9803A), 24 x Gigabit Ethernet 10/100/1000, 2 x SFP+',
+		'processors' => array ('procurve-25-to-26-combo-1000SFP','procurve-modular-1000T'),
+	),
+	'25506.11.1.82' => array
+	(
+		'dict_key' => 2176,
+		'text' => 'HP 1910 JE006A: 24 RJ-45/10-100-1000T(X) + 4 SFP-1000 ports',
+		'processors' => array ('procurve-25-to-26-combo-1000SFP','procurve-modular-1000T'),
```

Comment:
Ports 25-28 are SFP, but not combo; "procurve-modular-1000T" does not match "GigabitEthernet1/0/x"; most importantly, this OID is already supported (see Mantis#1293). I have closed Mantis#1601 as duplicate. Please remove this section.